### DISCUSSION TITLE
Learning partners!

### DISCUSSION THREAD

**Discussion by Silverbanks** [community/community#108225] [GitHub Education]
### Hi everyone,

Hi everyone, I need learning partners, anyone interested, lets go!

**Comment by murat-yasar** [comment]
Hi silverbanks,
you need to be more specific about what you want to learn!
- What is your goal? (do you want to learn a specific language or do you have a project idea)
- How do you want to achieve it? (have you decided on a schedule or a development stack)
- What do you expect from your partner? (to contribute a project, teaching you a language or mentoring, etc)
